enterprise/models.py

```diff
@@ -942,7 +954,12 @@ def contains_content_items(self, content_id_field_name, content_id_values):
             updated_content_filter[content_id_field_name] = content_id_values
             results = CourseCatalogApiServiceClient().get_search_results(updated_content_filter)
```

Comment:
Instead of filtering the results below, just add the `has_enrollable_seats` and `is_program_eligible_for_one_click_purchase` flags to the `updated_content_filter`.

Reply:
@douglashall do you mean that we should update the OrderDict `updated_content_filter` with flags e.g. 

```
updated_content_filter.update({
            "has_enrollable_seats": True,
            "is_program_eligible_for_one_click_purchase": True
}) ???
Do you think will that work as course and programs have different flags values in them. I think, It will not fetch the data. correct me If I am missing your point. 
```

Reply:
Ah, good point. I guess we can't do that unfortunately.